### PR TITLE
[examples] Rename logits to log_probs

### PR DESCRIPTION
**Pull request by desktable**
This is a minor fix for better code readability.

In examples/cora.py, the model outputs log probabilities, rather than logits (logarithm of odds ratios). This PR changes the variable name from `logits` to `log_probs`.

### CODE REVIEW

**Comment by rusty1s** [comment]
Awesome :)
